src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -565,7 +565,7 @@ private Collection<string> GetNextFromPathUsingWildcards(string command, out Pro
             }
 
             provider = null;
-            return null;
+            return new Collection<string>();
```

Comment:
Maybe you should use this if read only is ok.

https://github.com/PowerShell/PowerShell/blob/0dc216ee7c682a6c64b7cf6a70b4ccde85cec23d/src/System.Management.Automation/engine/Utils.cs#L1455

Reply:
This breaks functionality in the searcher.